### PR TITLE
X audio error

### PR DESCRIPTION
**Pull request by debugwand**
* add handling for error event that will show a message to users
* allow users to recover from a network failure resulting in an error that reloads the audio and starts it from where they left off
* also reset all the state when load media occurs

### CODE REVIEW

File: components/x-audio/src/components/Loading.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ export default ({expanded}) => {
 	return (
 		<div role="status" aria-live="polite" className={classNameMap('audio-player__status')}>
 			<div className={`o-loading o-loading--${foregroundColor} o-loading--mini ${classNameMap('audio-player__loader')}`}></div>
-			Loading
+			LOREM
```
Comment (benbarnett) [inline components/x-audio/src/components/Loading.jsx]: Accidental commit?

File: components/x-audio/src/components/index.jsx [hunk 2]
```diff
@@ -32,8 +34,9 @@ export const Audio = ({
 		{expanded && <div className={classNameMap('audio-player__info__image')}><img alt="dummy"/></div>}
 		<div className={classNameMap('audio-player__info__title')}>{title}</div>
 		<div className={classNameMap('audio-player__info__series-name')}>{expanded ? seriesName : `${seriesName}:`}</div>
-		{!expanded && loading ? null : <TimeRemaining currentTime={currentTime} duration={duration} expanded={expanded}/>}
-		{loading && <Loading expanded={expanded} />}
+		{!expanded && loading || error ? null : <TimeRemaining currentTime={currentTime} duration={duration} expanded={expanded}/>}
+		{!error && loading && <Loading expanded={expanded} />}
+		{error && <ErrorMessage />}
```
Comment (benbarnett) [inline components/x-audio/src/components/index.jsx]: This makes me wonder if we should split out the minimised and expanded views into separate components, and just pass them all the props. Perhaps something to consider after the release?

File: components/x-audio/src/redux/player-logic.js [hunk 3]
```diff
@@ -77,18 +85,19 @@ export const middleware = (store, audio = new Audio()) => {
...
+	// 	'loadstart',
+	// 	'progress',
+	// 	'canplay',
+	// 	'canplaythrough',
+	// 	'error'
+	// ].forEach(evtName => {
+	// 	audio.addEventListener(evtName, () => {
+	// 		console.log('audio eventy', evtName);
+	// 	});
+	// });
```
Comment (benbarnett) [inline components/x-audio/src/redux/player-logic.js]: Worth just killing this now?

Comment (debugwand) [reply components/x-audio/src/redux/player-logic.js]: might be useful for when we get onto tracking

File: components/x-audio/src/components/index.jsx (hunk 2 above)
Comment (debugwand) [reply components/x-audio/src/components/index.jsx]: would be easier to understand it

**Pull request merged**